sbpy/data/phys.py

```diff
@@ -12,8 +12,11 @@
 from collections import OrderedDict
 
 from numpy import ndarray, array, isnan, nan
+from scipy import interpolate
```

Comment:
scipy is an optional package.  This should be wrapped to catch an import error, or the import moved to the method that needs it.

Reply:
Taken care of within the method that needs it